contracts/test/Migrator.t.sol

```diff
@@ -1,22 +1,14 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity 0.8.7;
 
-import { IERC20 } from "../../modules/erc20/contracts/interfaces/IERC20.sol";
+import { Test } from "../../modules/forge-std/src/Test.sol";
 
-import { TestUtils } from "../../modules/contract-test-utils/contracts/test.sol";
+import { IERC20 }    from "../../modules/erc20/contracts/interfaces/IERC20.sol";
 import { MockERC20 } from "../../modules/erc20/contracts/test/mocks/MockERC20.sol";
 
 import { Migrator } from "../Migrator.sol";
 
-contract SomeAccount {
-
-    function approve(address token_, address spender_, uint256 amount_) external {
-        IERC20(token_).approve(spender_, amount_);
-    }
-
-}
-
-contract MigratorConstructorTest is TestUtils {
+contract MigratorConstructorTest is Test {
```

Comment:
Let's move this test folder and files out to have the repo reflect our current repos